apps/emqx_dashboard/src/emqx_dashboard_admin.erl

```diff
@@ -74,6 +75,28 @@ mnesia(boot) ->
         ]}
     ]).
 
+%%--------------------------------------------------------------------
+%% bootstrap API
+%%--------------------------------------------------------------------
+
+-spec add_default_user() -> {ok, map() | empty | default_user_exists} | {error, any()}.
+add_default_user() ->
+    add_default_user(binenv(default_username), binenv(default_password)).
+
+-spec add_bootstrap_user() -> ok | {error, _}.
+add_bootstrap_user() ->
+    case emqx:get_config([dashboard, bootstrap_user], undefined) of
+        undefined ->
+            ok;
+        File ->
+            case mnesia:table_info(?ADMIN, size) of
+                0 ->
+                    add_bootstrap_user(File);
```

Comment:
```suggestion
                    ?LOG(debug, "Add dashboard bootstrap users from ~s", [File]),
                    add_bootstrap_user(File);
```

Reply:
Do we need to add some logs to indicate whether we have added these users for this boot?